src/udp_wrap.cc

```diff
@@ -728,9 +728,12 @@ void UDPWrap::OnRecv(ssize_t nread,
     bs = BackingStore::Reallocate(isolate, std::move(bs), nread);
   }
 
+  Local<Object> address;
+  if (!AddressToJS(env, addr).ToLocal(&address)) return;
```

Comment:
This is a bit problematic, because nothing in the code here handles exceptions (since this is in a top-level callback from libuv). I think you’ll want to add a `TryCatch` here with `try_catch.SetVerbose(true)`, so that we get the typical uncaught-exception error messages in this case.

Reply:
Done!